web/js/modules/product-picker/selectors.js

```diff
@@ -0,0 +1,50 @@
+import {
+  sortBy as lodashSortBy,
+  indexOf as lodashIndexOf,
+} from 'lodash';
+import { createSelector } from 'reselect';
+import buildLayerFacetProps from './formatConfig';
+
+const decodeHtml = (html) => {
```

Comment:
maybe a good util function?

Reply:
I thought about it but since it's only used here (and I just broke it out of the existing function below that was moved from another file) I'd like to hold off for now.